packages/valory/skills/decision_maker_abci/tests/__init__.py

```diff
@@ -1,25 +0,0 @@
-# -*- coding: utf-8 -*-
-# ------------------------------------------------------------------------------
-#
```

Comment:
Would be good to keep that and add at least one test so we can later build on it. Rather than deleting it now and adding it back in later.

Reply:
Done in 73567cee9c1078618f896e95ae43e00f6941af87.